src/components/common/v2/IconButton.tsx

```diff
@@ -14,9 +14,18 @@ type IconButtonProps = {
 	iconName: keyof typeof Icn;
 	onClick: () => void;
```

Comment:
```suggestion
	onClick?: () => void;
```
disabled=false일때는 onClick이 안넘어가니까 옵셔널로 처리해주시면 감사하겠습니다!!

Reply:
감사합니당